rest-service/manager_rest/test/endpoints/test_labels.py

```diff
@@ -85,7 +85,7 @@ def test_remove_resource_labels(self):
         resource = self.put_resource_with_labels(self.LABELS)
         self.assert_resource_labels(resource.labels, self.LABELS)
         updated_res = self.update_resource_labels(resource.id, [])
-        self.assertEmpty(updated_res.labels)
+        assert not updated_res.labels
```

Comment:
assert updated_res.labels == [] might be better given that we were previously making sure it wasn't None?

Reply:
In fact, same with the others of this type- that maintains the assertions but shorter and clearer.